src/app/heroes/hero-detail/hero-detail.component.html

```diff
@@ -0,0 +1,11 @@
+<div *ngIf="hero">
+  <h2>{{hero.name | uppercase}} Details</h2>
+  <div><span><b>id: </b></span>{{hero.id}}</div>
+  <div>
+    <label for="hero-name">Hero name: </label>
+    <input id="hero-name" [(ngModel)]="hero.name" placeholder="Hero name"/>
+  </div>
+  <p><span><b>Weapon: </b></span>{{hero.weapon.name}} - <span><b>Dammage: </b></span>{{hero.weapon.damage}}</p>
+  <p><span><b>Armour: </b></span>{{hero.armour.name}} - <span><b>Health: </b></span>{{hero.armour.health}}</p>
```

Comment:
shouldn't these be editable too

Reply:
My mistake! I have update this by adding select for update weapon and armour for hero.